### DISCUSSION TITLE
MegaMenuModule Mouse blur event

### DISCUSSION THREAD

**Discussion by shahinrm** [primefaces/community#1836] [PrimeNG] [answered]
In Mega Menu, it is not possible to close the menu when we remove the mouse from the hovered area, I wish it would be possible to close it when the mouse is removed.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the valuable feedback, MegaMenu listens to click event to close the overlay. Unfortunately, there is no option to make it possible by hover. However, we'll wait for the upvotes to implement as a new feature.

**Reply by aliyeh78** [reply]
I need this feature please add this

**Reply by alihit96** [reply]
I've had the same issue like forever!!

**Comment by amirzeinaly** [comment]
This feature is essential for all developer

**Comment by shahinrm** [comment]
> Hi,
> 
> Thanks for the valuable feedback, MegaMenu listens to click event to close the overlay. Unfortunately, there is no option to make it possible by hover. However, we'll wait for the upvotes to implement as a new feature.


@cetincakiroglu I am really lame about this feature, is it possible to check it faster?
